install-deps.sh

```diff
@@ -314,6 +314,10 @@ else
                 ensure_decent_gcc_on_ubuntu 9 bionic
                 [ ! $NO_BOOST_PKGS ] && install_boost_on_ubuntu bionic
                 ;;
+            *Disco*)
```

Comment:
probably we don't need to install gcc-9 on ubuntu Disco. as it has GCC-8.3 already?

Reply:
Yes, Ubuntu Disco Dingo ships with GCC-8.3